optimade/validator/validator.py

```diff
@@ -116,7 +117,7 @@ def get(self, request: str):
                 self.response = requests.get(self.last_request)
             except requests.exceptions.ConnectionError:
                 sys.exit(f"No response from server at {self.last_request}")
-            except requests.exceptions.MissingField:
+            except requests.exceptions.MissingSchema:
```

Comment:
Just checking that this was replaced, and you can no longer get a `MissingField` exception here?

Reply:
Yeah, this was a result of a faulty change from `Schema` to `Field`, since this is what is used by `pydantic` now to descripte field values. So it was probably wrongly changed in a search-and-replace previously - I found it randomly and made an issue for it #129, which I have also cited in the OP with an edit.